R/transformAbundance.R

```diff
@@ -0,0 +1,400 @@
+#' Transform Abundance
+#'
+#' This function applies transformation to abundance table, which is returned.
+#' Additionally, the information can be directly stored in the \code{assay}.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object.
+#'
+#' @param abund_values
+#' A single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}}
+#'   to be transformed.
+#'
+#' @param transform
+#' A single character value for selecting the transformation method.
+#'
+#' @param name
+#' A single character value specifying the name of transformed abundance table.
+#'
+#' @param pseudocount
+#' Boolean or numeric value deciding whether pseudocount is added. Numerical
+#' value specifies the value of pseudocount. When selecting pseudocount==TRUE,
+#' pseudocount is 1.
+#'
+#' @param scale
+#' A numeric value for scaling. Values are multiplied with specified value.
```

Comment:
I think `scalingFactor` would be better to avoid any confusion with the case R function `scale` and its arguments

Reply:
scale --> scalingFactor